src/SDKs/CognitiveServices/management/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.csproj

```diff
@@ -6,18 +6,26 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.CognitiveServices</PackageId>
 		<Description>Microsoft Azure Management Cognitive Services Library</Description>
-		<Version>2.0.0</Version>
+		<Version>2.1.0</Version>
 		<AssemblyName>Microsoft.Azure.Management.CognitiveServices</AssemblyName>
 		<PackageTags>Microsoft Azure Cognitive Services management;Cognitive Services;Cognitive Services management</PackageTags>
-		<PackageReleaseNotes/>
+		<PackageReleaseNotes />
 	</PropertyGroup>
 	<PropertyGroup>
 		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
 	</PropertyGroup>
+	<ItemGroup>
+	  <Compile Remove="Properties\AssemblyInfo.cs" />
```

Comment:
@felixwa please remove from line 17 to 19
also remove line 24 to 27

Reply:
@shahabhijeet , I'm seeing CS0579 if AssemblyInfo.cs is included.
obj\Debug\netstandard1.4\Microsoft.Azure.Management.CognitiveServices.AssemblyInfo.cs(14,12,14,54): error CS0579: Duplicate 'System.Reflection.AssemblyCompanyAttribute' attribute

Can you please tell me how to mitigate this error? Should I add below?
  <PropertyGroup>
    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
  </PropertyGroup>

Reply:
I talked to @DavidLiCIG  and he said we can remove this AssemblyInfo.cs file so I removed it.